### ISSUE TITLE
Sort by language

### ISSUE THREAD

**Issue by picsi** [eloquence/lib.reviews#198]
As lib.reviews grows, it might become desirable to sort reviews by languages, perhaps in lists or even as a search option.

**Comment by leethax666** [comment]
Yeah, I love the Euros, but I can't read their reviews! I'd love to filter by language
